code/drasil-build/package.yaml

```diff
@@ -12,6 +12,9 @@ extra-source-files: []
 dependencies:
 - base >= 4.7 && < 5
 - pretty
+- text
+- word-wrap
+- drasil-gool
```

Comment:
I don't like this added dependency.

I see why this happened: someone noticed the concept duplication of `Comment` from GOOL and here. That's good. The solution was not so good: just use GOOL's notion.

The real solution is not notice that there is a common primitive concept `Comment` that perhaps multiple packages might want to share. So that concept should be moved to a (small?) (new?) package that defines these common concepts.

In fact, I'm pretty sure that's how `drasil-code-base` was born. [Turns out some things were put into it that should not have been, but that's a different thing!]

Reply:
Ok, I will remove that dependency and transfer the `Comment` type to another package on which GOOL and `drasil-build` can depend.

Reply:
While `drasil-utils` is an acceptable place (i.e. doesn't introduce bad dependencies), it's also not a great place. 'utils' has no meaning (it's a catch-all), so this doesn't increase the "semantic content" of `Comment`.

Please introduce `drasil-codelang` for this. However, I would be fine if you preferred to do this in a separate PR - just let me know.

Reply:
I will introduce that right now!